src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -218,10 +218,10 @@ class VtolAttitudeControl
 	int _transition_command;
 	bool _abort_front_transition;
 
-	VtolType *_vtol_type;	// base class for different vtol types
-	Tiltrotor *_tiltrotor;	// tailsitter vtol type
-	Tailsitter *_tailsitter;	// tiltrotor vtol type
-	Standard *_standard;	// standard vtol type
+	VtolType *_vtol_type = nullptr;	// base class for different vtol types
+	Tiltrotor *_tiltrotor = nullptr;	// tailsitter vtol type
+	Tailsitter *_tailsitter = nullptr;	// tiltrotor vtol type
+	Standard *_standard = nullptr;	// standard vtol type
```

Comment:
This is a nitpick unrelated to your change, but is there a reason we need these? The single _vtol_type should be sufficient.